modules/sandboxed-containers-building-blocks.adoc

```diff
@@ -4,4 +4,8 @@
 
 [id="sandboxed-containers-building-blocks_{context}"]
 
-= Sandboxed containers building blocks
+= {sandboxed-containers-first} building blocks
+
+The {sandboxed-containers-operator} encapsulates all of the components from Kata containers. It manages installation, lifecycle, and configuration tasks.
+
+The {sandboxed-containers-operator} uses the link:https://quay.io/isolatedcontainers/sandboxed-containers-operator[sandboxed-containers-operator] container image. The image contains the controller part of the Operator that monitors and manages the `KataConfig` resource.
```

Comment:
@zanetworker AFAICS it will be registry.redhat.io/rh-osbs/openshift-tech-preview/sandboxed-containers-operator-bundle and registry.redhat.io/rh-osbs/openshift-tech-preview/sandboxed-containers-operator

Reply:
@jensfr, @zanetworker - Here are we trying to tell the user where they can download the Operator? Or are we trying to tell them the parts that make up the operator?

Reply:
Its closer to the latter, basically how the operator deploys its content.

Reply:
Rewrite suggestion: The {sandboxed-containers-operator} is constructed from the [sandboxed operator bundle](registry.redhat.io/rh-osbs/openshift-tech-preview/sandboxed-containers-operator-bundle) and the [container image](registry.redhat.io/rh-osbs/openshift-tech-preview/sandboxed-containers-operator). The images contains the controller part of the Operator that monitors and manages the `KataConfig` resource.

Does that sound write? @zanetworker @jensfr  -- Feel free to offer suggestions for rework.

Reply:
How about: `The {sandboxed-containers-operator} is packaged in the  [Operator bundle format](https://docs.openshift.com/container-platform/4.7/operators/operator_sdk/osdk-working-bundle-images.html). Therefore it comes in two container images. The Bundle image contains metadata and is required to make the operator OLM-ready. The second container image contains the actual controller that monitors and manages the KataConfig resource.`

Reply:
I like that wording.

Reply:
Made the changes based on @jensfr comment.